src/ui/widgets/PitServiceWidget/PitApproachRail/PitApproachRail.module.scss

```
@@ -0,0 +1,188 @@
+
+.urgencybrake {
+  --rail-accent: #{$race-red};
+
+  animation: railPulse 0.4s steps(2, end) infinite;
+}
+
+.urgencyarrived {
+  --rail-accent: #{$race-green};
+}
+
+@keyframes railPulse {
```

Stylelint (17.14.0)

[error] Expected keyframe name "railPulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)